apps/web/src/app/_components/landing-page/_index.scss

```
@@ -247,8 +202,18 @@
       align-items: center;
       @apply text-white;
       border-radius: 30px;
-      border: none;
+      border: 2px solid $blue;
       outline: none;
+      font-weight: 600;
+      transition: opacity 0.15s ease;
+      &:hover {
+        opacity: 0.9;
+      }
+      // Secondary CTA: outlined, lets the filled primary dominate the hierarchy.
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)